### ISSUE TITLE
placeholder for security issue

### ISSUE THREAD

**Issue by rsc** [golang/go#22131]
Dup of #22125 but for Go 1.9.1.

**Comment by shurcooL** [comment]
> Fixed in Go 1.9.1 by [CL 68022](https://golang.org/cl/68022) (bc907974a35a).

Can you please clarify what commit that is? I'm not able to find "bc907974a35a".

**Comment by ianlancetaylor** [comment]
@shurcooL Doesn't the CL link work for you?  https://golang.org/cl/68022.

**Comment by shurcooL** [comment]
Yes it does, but it says it was merged as a different commit, a39bcecea6660d3c6d9770516df441c3f8fc47f5. That's why I am puzzled about what "bc907974a35a" is.

**Comment by ianlancetaylor** [comment]
Ah.  Perhaps a typo.

The change was committed three times that I know of, to different branches:

- a4544a0f8af001d1fb6df0e70750f570ec49ccf9
- a39bcecea6660d3c6d9770516df441c3f8fc47f5
- ec71ee078fd3243b78c0d404c8634bd97e38d7eb

**Comment by shurcooL** [comment]
> Ah. Perhaps a typo.

I see. Thank you for clarifying.
